package.json

```diff
@@ -69,10 +69,15 @@
 	"dependencies": {
 		"@babel/runtime": "^7.17.0",
 		"@supabase/supabase-js": "^1.31.1",
+		"bootstrap": "^5.1.3",
 		"express": "^4.17.2",
 		"helmet": "^5.0.2",
+		"jquery": "^3.6.0",
 		"morgan": "^1.10.0",
 		"pg": "^8.7.3",
+		"popper.js": "^1.16.1",
+		"react-bootstrap": "^2.2.0",
+		"react-datepicker": "^4.7.0",
```

Comment:
These packages only seem to be used by the React client build, therefore should be development **not** production dependencies. Production dependencies are only those used by the Express server at runtime.

Reply:
Thank you, I have removed these dependencies and put them in the devDependencies for now.